libraries/cmd/dock_utils.cpp

```diff
@@ -24,18 +24,22 @@
 
 #include "dock_utils.h"
 
+#include "universe.h"
```

Comment:
`universe.h` is a large file. We should strive to make our files as encapsulated as possible. 
In this case, if you need `isPlayerStarship`, use DI instead.
Ideally, we could remove `Unit` and just keep `damage/core_vector` and have a nice, self contained unit to test.

Reply:
Good point. Yeah, I don't think I need universe.h at all now, since I've removed the logging statements that only fired when `isPlayerStarship` was true.